### PR TITLE
working on table and pagination

### PR DESCRIPTION
**Pull request by GAYATRIBHAGWAT11**
working on table and pagination

### CODE REVIEW

File: src/Component/ContextApi/ContextApi.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+const createContextApi = createContext();
+
+function ContextApi({ children }) {
+  const [coviddata, setCovidData] = useState([]);
+  const [error, setError] = useState("");
+  const [showToast, setShowToast] = useState(false);
+  useEffect(() => {
+    axios
+      .get(
+        `https://vaccovid-coronavirus-vaccine-and-treatment-tracker.p.rapidapi.com/api/npm-covid-data/`,
```
Comment (Souvikdeb2612) [inline src/Component/ContextApi/ContextApi.js:14]: store url separately in a variable/state and use wherever required

File: src/Component/CovidGraph/CovidGraph.jsx [hunk 2]
```diff
@@ -1,77 +1,52 @@
-import React, { useState, useEffect } from "react";
-import axios from "axios";
+import React from "react";
+import { useContext } from "react";
+import { createContextApi } from "../ContextApi/ContextApi";
+import "../CovidGraph/CovidGraph.css";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
```
Comment (Souvikdeb2612) [inline src/Component/CovidGraph/CovidGraph.jsx]: First external imports occur that is imports from installed libraries, then 1 line space, then all internal imports that is files/components/functions that you have created, then 1 line space and then finally all css imports.

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/Component/DataTable/Posts.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+                            <th>Active Cases</th>
+                            <th>cases(per million)</th>
+                            <th>Total cases</th>
+                            <th>Total Deaths</th>
+                            <th>Total Recovered</th>
+
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {posts.map(post => {
```
Comment (Souvikdeb2612) [inline src/Component/DataTable/Posts.jsx]: destructure post here

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

**Comment by GAYATRIBHAGWAT11** [comment]
final changes and registration form updated

**Comment by GAYATRIBHAGWAT11** [comment]
changing reviewer to hema

**Pull request merged**